components/api/db/cellery-hub-db.bal

```diff
@@ -21,16 +21,22 @@ import ballerina/mysql;
 import ballerina/sql;
 import cellery_hub_api/gen;
 
-public function getOrganization (string name) returns json|error {
-    log:printDebug("Performing organization search for Org name in REGISTRY_ORGANIZATION table " + name);
-    table<record {}> res =  check connection->select(GET_ORG_QUERY,gen:OrgResponse,name, loadToMemory = true);
+public function getOrganization (string orgName) returns json|error {
+    log:printDebug("Performing search on REGISTRY_ORGANIZATION table, Org name : " + orgName);
```

Comment:
This is not a search. This is simply fetching a single organization. Please update log